web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -62,4 +81,56 @@ describe("<SidebarCardsList />", () => {
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_cards).toHaveBeenCalledTimes(1);
   });
+
+  describe("empty state messaging", () => {
+    it("displays Complete Required Tasks Message when there are no primary actions or opportunities", () => {
```

Comment:
```suggestion
    it("displays Complete Required Tasks Message when there are no primary actions and opportunities are not displayed", () => {
```
[sand] Reading this with fresh eyes, I'm thinking about what this test name is conveying. While the message is true, the key piece of logic has to do with `displayCertifications` and `displayFundings` not specifically the opportunity cards being present.